kratos/mpi/tests/test_mpi_processes.py

```diff
@@ -0,0 +1,125 @@
+﻿import os
+import sys
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics
+import KratosMultiphysics.MetisApplication as KratosMetis
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+from KratosMultiphysics.mpi import distributed_import_model_part_utility
+import json
+
+def GetFilePath(fileName):
+    return os.path.dirname(os.path.realpath(__file__)) + "/" + fileName
+
+class TestMPIProcesses(KratosUnittest.TestCase):
+
+    def _ReadSerialModelPart(self,model_part, mdpa_file_name):
+        import_flags = KratosMultiphysics.ModelPartIO.READ | KratosMultiphysics.ModelPartIO.SKIP_TIMER
+        KratosMultiphysics.ModelPartIO(mdpa_file_name, import_flags).ReadModelPart(model_part)
+
+    def _ReadDistributedModelPart(self,model_part, mdpa_file_name):
+
+        importer_settings = KratosMultiphysics.Parameters("""{
+            "model_import_settings": {
+                "input_type": "mdpa",
+                "input_filename": \"""" + mdpa_file_name + """\",
+                "partition_in_memory" : true
+            },
+            "echo_level" : 0
+        }""")

```

Comment:
this is solved better now here: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/testing/utilities.py#L14